en/micro-integrator/docs/references/connectors/kafka-connector/kafka-connector-configuration.md

```diff
@@ -0,0 +1,131 @@
+# Kafka Connector Configuration
+
+[[Prerequisites]](#Prerequisites) [[Initializing the Connector]](#initializing-the-connector)
+
+The Kafka connector allows you to access the Kafka Producer API through WSO2 EI and acts as a message producer that facilitates message publishing. The Kafka connector sends messages to the Kafka brokers.
+
+Kafka is a distributed publish-subscribe messaging system that maintains feeds of messages in topics. Producers write data to topics and consumers read from topics. For more information on Apache Kafka, see [Apache Kafka documentation](http://kafka.apache.org/documentation.html).
```

Comment:
Kafka is a distributed publish-subscribe messaging system that maintains the feeds of

Reply:
Updated the content of the first paragraph.